### PR TITLE
Add Hivie to `osu! wiki maintainers`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by Walavouchey — COMMENTED** [review]
translations should be updated too

File: wiki/People/osu!_wiki_maintainers/ru.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 | :-- | :-- | :-- | :-- |
 | ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) | [cl8n](https://github.com/cl8n) | Неактивен |  |
 | ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | [Ephemeralis](https://github.com/Ephemeralis) | **Активен** |  |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | [Hiviexd](https://github.com/Hiviexd) | **Активен** | Арабский |
```
Comment (agatemosu) [inline wiki/People/osu!_wiki_maintainers/ru.md:17]:
```suggestion
| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | [Hiviexd](https://github.com/Hiviexd) | **Активен** | Ара́бский |
```

Comment (minusQuantumNeko) [reply wiki/People/osu!_wiki_maintainers/ru.md:17]: ok, this one is just wrong xD

Comment (agatemosu) [reply wiki/People/osu!_wiki_maintainers/ru.md:17]: okay

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
for language names just use cldr like group-info. also I assume it was just mistake here but avoid using your "master" branch for prs

**Pull request merged**